src/compiler/scala/reflect/reify/phases/Metalevels.scala

```diff
@@ -128,13 +128,16 @@ trait Metalevels {
     // A: nothing. reified trees give us problems because they sometimes create dimensional rifts as described above
     //    to the contrast, reified types (i.e. synthetic typetags materialized by Implicits.scala) always stay on the same metalevel as their enclosing code
     override def transform(tree: Tree): Tree = tree match {
-      case TreeSplice(ReifiedTree(universe, mirror, symtab, rtree, tpe, rtpe, concrete)) =>
-        if (reifyDebug) println("entering inlineable splice: " + tree)
-        val inlinees = symtab.syms filter (_.isLocalToReifee)
-        inlinees foreach (inlinee => symtab.symAliases(inlinee) foreach (alias => inlineableBindings(alias) = symtab.symBinding(inlinee)))
+      case TreeSplice(ReifiedTree(_universe, _mirror, symtab, _rtree, _tpe, _rtpe, _concrete)) =>
```

Comment:
This is where I decided to use underscore as special marker. Only `symtab` is "used" and replaced. Maybe the better idiom is `case rt @ ReifiedTree(_, ..., symtab, ...) => rt.copy(symtab = symtab1)`. I was deterred by `withinSplice` etc. I didn't want to change anything but syntax.